src/build/post-build.js

```diff
@@ -1,11 +1,29 @@
-const execSync = require('child_process').execSync;
+const path = require('node:path');
+const execSync = require('node:child_process').execSync;
 const chalk = require('chalk');
+const fs = require('node:fs')
 
 execSync('git checkout src/app/buildtimestamp.ts');
 execSync('git checkout ngsw-config.json');
 execSync('npx sentry-cli sourcemaps inject ./dist/');
 execSync('npx ngsw-config ./dist ngsw-config.json "/app/"');
 
+const ngswPath = path.join(__dirname, '../../dist/ngsw.json')
+const ngswJSON = require(ngswPath);
+
+const build_time = new Date().toJSON();
```

Comment:
In the pre-build.js we also create a build_time. I don't believe it to be an issue that they differ. Agred @castaway?